<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Terreno</class>
 <widget class="QWidget" name="Terreno">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Terrain&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="3">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>GRS</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QComboBox" name="grsComboBox">
          <item>
           <property name="text">
            <string>SIRGAS2000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SAD69</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>WGS84</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QComboBox" name="cpsComboBox">
          <item>
           <property name="text">
            <string>UTM</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>CPS</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>UTM Fuse</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QSpinBox" name="utmFuseSpin">
          <property name="specialValueText">
           <string/>
          </property>
          <property name="maximum">
           <number>60</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="br::uerj::eng::efoto::EfotoDoubleSpinBox" name="meanAltSpinBox">
          <property name="toolTip">
           <string>Mean Altitude Above Sea Level</string>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="suffix">
           <string> m</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-1000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="meanAltitudeLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Mean Altitude</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="maxAltitudeLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Max Altitude</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="br::uerj::eng::efoto::EfotoDoubleSpinBox" name="maxAltSpinBox">
          <property name="toolTip">
           <string>Max Altitude Above Sea Level</string>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string> m</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>10000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="br::uerj::eng::efoto::EfotoDoubleSpinBox" name="minAltSpinBox">
          <property name="toolTip">
           <string>Min Altitude Above Sea Level</string>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="suffix">
           <string> m</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-1000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="minAltitudeLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Min Altitude</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>246</width>
          <height>185</height>
         </size>
        </property>
        <property name="title">
         <string>Approximate geographic coordinates of center project area</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Latitude</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="sizeConstraint">
               <enum>QLayout::SetMaximumSize</enum>
              </property>
              <item>
               <widget class="br::uerj::eng::efoto::DmsEdit" name="latDmsEdit">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>323</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QComboBox" name="latDirCombo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="sizeAdjustPolicy">
                 <enum>QComboBox::AdjustToContents</enum>
                </property>
                <item>
                 <property name="text">
                  <string>Northern Hemisphere</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Southern Hemisphere</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Longitude</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="br::uerj::eng::efoto::DmsEdit" name="longDmsEdit">
                <property name="maximumSize">
                 <size>
                  <width>323</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QComboBox" name="longDirCombo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>West of Greenwich</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>East of Greenwich</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>br::uerj::eng::efoto::EfotoDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>EfotoDoubleSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>br::uerj::eng::efoto::DmsEdit</class>
   <extends>QLineEdit</extends>
   <header>DmsEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>maxAltSpinBox</tabstop>
  <tabstop>minAltSpinBox</tabstop>
  <tabstop>meanAltSpinBox</tabstop>
  <tabstop>grsComboBox</tabstop>
  <tabstop>cpsComboBox</tabstop>
  <tabstop>utmFuseSpin</tabstop>
  <tabstop>latDirCombo</tabstop>
  <tabstop>longDirCombo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
